### PR TITLE
[CHLO] Add erf_inv

### PR DESCRIPTION
**Pull request by atondwal**

### CODE REVIEW

**Comment by atondwal** [comment]
This ports over the change from tf https://github.com/tensorflow/tensorflow/blob/master/third_party/stablehlo/temporary.patch

File: stablehlo/dialect/ChloOps.td [hunk 1]
```diff
@@ -673,6 +673,15 @@ def CHLO_ErfOp : CHLO_UnaryElementwiseOp<"erf",
...
 
+def CHLO_ErfInvOp : CHLO_UnaryElementwiseOp<"erf_inv",
+    [HLO_CompatibleOperandsAndResultType], HLO_FpTensor, HLO_FpTensor> {
+  let summary = "Inverse Erf";
+  let description = [{
+    Returns `ErfInv(operand)` element-wise.
+  }];
+}
+
+
```
Comment (GleasonK) [inline stablehlo/dialect/ChloOps.td:684]: Remove extra space

File: stablehlo/dialect/ChloOps.td [hunk 2]
```diff
@@ -673,6 +673,15 @@ def CHLO_ErfOp : CHLO_UnaryElementwiseOp<"erf",
   }];
 }
 
+def CHLO_ErfInvOp : CHLO_UnaryElementwiseOp<"erf_inv",
+    [HLO_CompatibleOperandsAndResultType], HLO_FpTensor, HLO_FpTensor> {
```
Comment (GleasonK) [inline stablehlo/dialect/ChloOps.td:677]: Does this op support anything other than f32/f64? Should we verify type at all?

**Review by GleasonK — APPROVED** [review]

File: stablehlo/dialect/ChloOps.td (hunk 2 above)
Comment (atondwal) [reply stablehlo/dialect/ChloOps.td:677]: In practice the lowering in mlir_hlo upcasts less wide types, but in this repo I don't think it makes sense to restrict it. Also, leaving it like this is consistent with the other CHLO ops

File: stablehlo/dialect/ChloOps.td (hunk 1 above)
Comment (atondwal) [reply stablehlo/dialect/ChloOps.td]: Done

**Pull request merged**